File: week_1_assignment/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = get_int("Height: ");
    while (height < 1)
    {
        height = get_int("Height: ");
    }

    for (int i = 1; i < height + 1; i++)
    {
        for (int k = 1; k < height - i + 1; k++)
        {
            printf(" ");
        }
        for (int j = 1; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week_6_assignment/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(1, height+1):
    for j in range(1, height-i+1):
        print(" ", end="")
    for k in range(1, i+1):
        print("#", end="")
    print("  ", end="")
    for a in range(1, i+1):
        print("#", end="")
    print()
